decred/decrediton
# Add update version available check

To test:  
- change version to anything below v1.1.3 in app/package.json
- if an update is shown to be available then it will appear a button on the launcher page (next to settings/logs in the upper right corner)
